### PR TITLE
Multiple style updates and textColor option for hero

### PR DESCRIPTION
**Pull request by priceba**

### CODE REVIEW

**Review by mrewers — APPROVED** [review]
Looks okay. I made a number of changes, please review the comments I made on your PR.

Main thing that sticks out is that you broke encapsulation in a couple of places. Putting the state-specific styles requested by Tianna directly into the component CSS. This makes the plugin less generic and adaptable. While it is harder, whenever possible we should limit site specific styling to the state.scss file.

Also, not sure if it was intentional or not, but you didn't compile a prod version of the JS/CSS so none of these changes would take effect if the plugin was installed. While the build process will automatically run when the plugin is versioned, I still recommend occasionally recompiling the bundles and pushing them up when adding several changes.

File: public/blocks/hero/Hero/Hero.js [hunk 1]
```diff
@@ -19,11 +19,13 @@ const Hero = ( { id } ) => {
       files,
       lines,
       subtitle,
+      textColor,
       title,
       type,
     } = meta;
 
     const titleAlignment = align === 'center' || align === 'title' ? 'center' : 'left';
+    // const txtColor = color === 'lighter' || color === 'darker' ? '#ffffff' : '#333333';
```
[the remark this answers is not in the document]
Comment (priceba) [reply public/blocks/hero/Hero/Hero.js]: Thanks!

File: admin/metabox/ajax/sanitizers/class-sanitize-hero-meta.php [hunk 2]
```diff
@@ -44,6 +44,10 @@ public function sanitize_inputs( $data, $uploads ) {
       $sanitized['buttons'] = $sanitize_buttons->sanitize_buttons( $unsanitary['buttons'] );
     }
 
+    if ( ! empty( $unsanitary['textColor'] ) ) {
+      $sanitized['textColor'] = sanitize_text_field( $unsanitary['textColor'] );
+    }
+
```
[the remark this answers is not in the document]
Comment (priceba) [reply admin/metabox/ajax/sanitizers/class-sanitize-hero-meta.php]: Noticed you did that that was an error. Thanks for the correction.

File: public/blocks/stats/Stats/Stats.module.scss [hunk 3]
```diff
@@ -33,19 +33,24 @@
...
     text-decoration: none;
     text-align: center;
     margin-top: 26px;
     padding-bottom: 1rem;
     width: 300px;
+    
+    > span {
+      padding-left: 0.4rem;
+      display: inline-block;
+      line-height: 3.85rem;
```
[the remark this answers is not in the document]
Comment (priceba) [reply public/blocks/stats/Stats/Stats.module.scss]: Will remember that!

File: public/blocks/text/Text/Text.module.scss [hunk 4]
```diff
@@ -62,6 +62,10 @@
   width: 100%;
 }
 
+.gpalab-text .gpalab-normalizer > div {
+  min-height: none;
+}
+
```
[the remark this answers is not in the document]
Comment (priceba) [reply public/blocks/text/Text/Text.module.scss]: Referenced this file but do not recall changing it. Noted: Should have been removed from PR.

File: public/blocks/hero/Hero/Hero.module.scss [hunk 5]
```diff
@@ -1,5 +1,7 @@
 @import 'styles/colors.scss';
 
+$fontContent: 'Open Sans', Arial, sans-serif;
```
[the remark this answers is not in the document]
Comment (priceba) [reply public/blocks/hero/Hero/Hero.module.scss]: I tried placing this in state but could not call the scss variable. Was going to suggest creating a font component? But it appears to have worked somehow. I am going to go back and try this one again. Thanks!

**Pull request merged**